docs/python/examples/client.py

```diff
@@ -0,0 +1,40 @@
+import json
+import requests
+
+# NOTE: ollama must be running for this to work, start the ollama app or run `ollama serve`
+model = 'llama2' # TODO: update this for whatever model you wish to use
+context = [] # the context stores a conversation history, you can use this to make the model more context aware
+
+def generate(prompt):
```

Comment:
This should return the context such that the caller can pass it back if they want. It also avoids the use of `global` which is always a good idea

```python
def generate(prompt, context=[]);
  r = requests.post('...', json={'model': model, 'prompt': prompt, 'context': context})
  ...
  return body['context']

def main():
  context = []
  while True:
    ...
    context = generate(user_input, context)
```